src/libraries/System.Speech/src/Internal/GrammarBuilding/GrammarBuilderDictation.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Speech.Internal.SrgsParser;
+
+namespace System.Speech.Internal.GrammarBuilding
+{
+    internal sealed class GrammarBuilderDictation : GrammarBuilderBase
+    {
+        #region Constructors
+
+        internal GrammarBuilderDictation()
+            : this(null)
+        {
+        }
+
+        internal GrammarBuilderDictation(string category)
+        {
+            _category = category;
+        }
+
+        #endregion
+
+        #region Public Methods
+        public override bool Equals(object obj)
+        {
+            GrammarBuilderDictation refObj = obj as GrammarBuilderDictation;
```

Comment:
Is this a direct port? Are you fixing style, cause some of these things could be simplified.

Reply:
Yup, I think this one falls in the category that I don't think is worth churning. Thanks for pointing out.